Dockerfile

```diff
@@ -1,14 +1,28 @@
+# syntax=docker/dockerfile:1
 FROM golang:1.18 AS build
 
+WORKDIR /app
+COPY . .
 RUN apt-get update -y && apt-get install -y xz-utils
-RUN useradd -u 1000 -m esm
-RUN mkdir /esm && chown esm:esm /esm
-RUN git clone https://github.com/esm-dev/esm.sh /esm/esm.sh
-RUN git checkout v124
+RUN go build -o /esmd
 
-USER esm
-WORKDIR /esm
-RUN go build -o bin/esmd esm.sh/main.go
+FROM node:18-alpine3.16
+ENV USER_ID=65535
```

Comment:
just wondered, can we remove the `USER_ID` and `GROUP_ID ` perm?

Reply:
If you want the script to run as root, you can.

You can pass the `-u 0:0` params to docker.
That will be run as root.

That full command is: `docker run --rm -it  -p 8077:80 -u 0:0  ghcr.io/zsnmwy/esmd:latest`.

That's a security question.
See https://docs.docker.com/engine/security/rootless/